### ISSUE TITLE
Very Slow Run time on A Pre Trained Network - Inception V1

### ISSUE THREAD

**Issue by ugionet** [tensorflow/tensorflow#12949]
------------------------

### System information
- **example script (Very Similar label_Image.py but simpler)**:
- **Windows 10 X64 - TensorFlow installed from source **:
- **TensorFlow version (1.3)**:
- **Python version - 3.5.3**: 
- **Cmake 3.8.1**:
- **CUDA/cuDNN version - CUDA 8.0/ cuDNN 6.1**:
- **GPU model and memory - Nvidia GTX 1050, 4GB**:
- **Exact command to reproduce**:

### Describe the problem

I encountered a disturbing issue regarding the run time for processing an arbitrary image through an inception V1/GoogleNet pre-trained model the runtime took approximately 9.2 ms

When using the same image and exactly the same model with Caffe, It was running X4 faster than Tensorflow (2.3 ms).  The models  I used (both for Caffe and Tensorflow)  : 
https://github.com/beniz/deepdetect/issues/89
Labels file is attached. 

How is it even possible ?!?!  
Am I doing something incorrectly ? 

By the way, I carefully checked (with GPU-Z) and it seems Tensorflow  properly uses the GPU resources.

### Source code / logs ###
Attahced, along with the used label file. 
In order to use the (python) script, simply put it in the same folder as the model and label file., and rename it to :  TF_Running_Inception.py

[TF_Running_Inception.txt](https://github.com/tensorflow/tensorflow/files/1290863/TF_Running_Inception.txt)
[imagenet_slim_labels.txt](https://github.com/tensorflow/tensorflow/files/1290864/imagenet_slim_labels.txt)

**Comment by reedwm** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!

More specifically, you should ask on StackOverflow when debugging performance problems with your own script. If you find a performance issue with a TensorFlow op, model or any other specific part of Tensorflow, then file an issue. Try [tf_cnn_benchmarks](https://github.com/tensorflow/benchmarks/tree/master/scripts/tf_cnn_benchmarks) if you're benchmarking.
